plugin-locationcomponent/src/test/java/com/mapbox/maps/plugin/locationcomponent/LocationIndicatorLayerRendererTest.kt

```diff
@@ -163,7 +163,7 @@ class LocationIndicatorLayerRendererTest {
         any()
       )
     }
-    verify {
+    verify(exactly = 0) {
```

Comment:
why did it happen? looks a bit strange

Reply:
showBearingImage default is false, so the bearing image will not be added.

Reply:
Isn't it breaking change though?

Reply:
IMO no, we didn't break any API and the add image part is inside of SDK.

Reply:
I mean behaviour change, do we show the bearing image by default now?

Reply:
Yes, we are using a white circle as the bearing image, I'm not sure whether it's a bug. @pengdev Is it intentional?

Reply:
It's intentional to not show the heading arrow in the default setup. We used white circle as the bearing image as it doesn't contain the arrow image. If we bundle the bearing as default puck, then we probably need to move the white circle as the shadow image. and use the arrow as the bearing image.

Reply:
just checked the screen recording in the PR description, seems the puck style has removed the white circle, which probably is not correct and doesn't match the appearance of previous default 2D puck. Could you reference to the v9 puck implementation? @Chaoba